brave/src/main/java/com/linecorp/armeria/server/brave/ServiceRequestContextAdapter.java

```diff
@@ -70,13 +68,6 @@ public String method() {
             return ctx.method().name();
         }
 
-        /**
-         * Original implementation is calling {@link HttpServerAdapter#url(Object)} which needs {@link
-         * RequestLog#scheme()}, but because {@link RequestLog#scheme()} is not always available, we need to
-         * use {@link RequestContext#path()} directly.
-         *
-         * @see brave.http.HttpServerRequest#path()
-         */
```

Comment:
Unrelated with this PR, but this removes a deprecation warning during compilation. The comment block seems not relevant anymore.